src/wranglerAction.ts

```diff
@@ -0,0 +1,455 @@
+import {
+	debug,
+	getMultilineInput,
+	endGroup as originalEndGroup,
+	error as originalError,
+	info as originalInfo,
+	startGroup as originalStartGroup,
+	setFailed,
+	setOutput,
+} from "@actions/core";
+import { getExecOutput } from "@actions/exec";
+import semverEq from "semver/functions/eq";
+import { z } from "zod";
+import { exec, execShell } from "./exec";
+import { PackageManager } from "./packageManagers";
+import { semverCompare } from "./utils";
+import { getDetailedPagesDeployOutput } from "./wranglerArtifactManager";
+
+export type wranglerActionConfig = z.infer<typeof wranglerActionConfig>;
+export const wranglerActionConfig = z.object({
```

Comment:
Types are typically PascalCase

```suggestion
export type WranglerActionConfig = z.infer<typeof WranglerActionConfig>;
export const WranglerActionConfig = z.object({
```